WriteVHDLSyntax.py

```diff
@@ -478,14 +474,16 @@ def writeTopLevelMemoryType(mtypeB, memList, memInfo, extraports, delay = 0, spl
             wirelist += "t_"+mtypeB+"_ADDR"+disk+";\n"
             wirelist += "  signal "+mem+"_din_delay         : "
             wirelist += "t_"+mtypeB+"_DATA;\n"
-            if (interface != -1 and not extraports):
+            if (interface != -1 and not extraports) or (split == 1 and "TPAR" in mem):
+                if "TPAR" in mem:
+                    wirelist += "  signal "+mem+"_dummy   : std_logic_vector(1 downto 0);\n"
```

Comment:
@jasonfan393 Would it be possible to remove this signal by setting the corresponding address ports to `open`?
```suggestion
            if (interface != -1 and not extraports):
```